src/index.html

```diff
@@ -11,7 +11,170 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form__part">
+        <legend>Personal information</legend>
+
+        <label class="form__item">
+          Surname:
+          <input
+            type="text"
+            name="lastName"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form__item">
+          Name:
+          <input
+            type="text"
+            name="firstName"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form__item">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form__item">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+          >
+        </label>
+
+        <label class="form__item">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form__part">
+        <legend>Registration</legend>
+
+        <label class="form__item">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="form__item">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="32"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form__part">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form__item">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="loveCats"
+              value="y">
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="loveCats"
+              value="n"
+            >
+            No
+          </label>
+        </div>
+
+        <label class="form__item">
+          What is your favorite color?
+          <input
+            type="color"
+            name="favoriteColor"
+            >
```

Comment:
pay attention to indent
```suggestion
          >
```